server/server_gamedata.cpp

```diff
@@ -12,6 +14,22 @@ server_staticdata &server_staticdata::get() {
 
 server_staticdata::server_staticdata() {
     // read from postgres here
+    auto connection_info = ozo::make_connection_info("dbname=niba user=postgres");
```

Comment:
this looks good
we can maybe store the connection string as a static const somewhere